src/main/java/com/alkemy/ong/service/TestimonialService.java

```diff
@@ -57,6 +63,29 @@ public void deleteTestimonial(String id) {
              throw new ParamNotFound("The testimony is not found in the database.");
         }
     }
+    //Pagination of 10
+    @Transactional
+    public PagesDto<TestimonialDtoFull> getAllForPages(Integer page) {
```

Comment:
What about searchPaginatedTestimonials for method name?

Reply:
Done!